### PR TITLE
[Merged by Bors] - TY-3087 install rust targets via just

### PR DESCRIPTION
**Pull request by janpetschexain**
**References**

- [TY-3087]

**Summary**

- use full target triples for `cargo ndk`
- add rust targets to `just rust-deps`

[TY-3087]: https://xainag.atlassian.net/browse/TY-3087?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -12,7 +12,7 @@ FLUTTER_EXAMPLE_WORKSPACE=discovery_engine_flutter/example
 DART_VERSION=2.17.3
 FLUTTER_VERSION=3.0.2
 
-ANDROID_TARGETS="arm64-v8a x86_64 x86"
+ANDROID_TARGETS="aarch64-linux-android x86_64-linux-android i686-linux-android"
```
Comment (janpetschexain) [inline .env:15]: our github actions also contain the `armv7-linux-androideabi` target. we should either add it here as well or remove it there, what's the current situation?

Comment (acrrd) [reply .env:15]: We usually don't want to build for it locally while we want it on the CI.

Comment (janpetschexain) [reply .env:15]: ah ok, then i'll leave it as is 👌

File: justfile [hunk 2]
```diff
@@ -46,7 +46,15 @@ flutter-deps:
 
 # Fetches rust dependencies
 rust-deps:
-    cd "$RUST_WORKSPACE"; \
+    #!/usr/bin/env bash
+    set -eux -o pipefail
+    cd "$RUST_WORKSPACE"
+    for TARGET in $ANDROID_TARGETS; do
+        rustup target add $TARGET
+    done
```
Comment (acrrd) [inline justfile:54]: :art: We could have a `if os() == "macos"`

**Review by acrrd — APPROVED** [review]

**Comment by janpetschexain** [comment]
bors merge

**Pull request closed without merging**